db/migrate/20200630070422_require_timestamps.rb

```diff
@@ -0,0 +1,253 @@
+class RequireTimestamps < ActiveRecord::Migration
+  def up
+    current_time = Time.now
```

Comment:
Have you checked what's the actual value that the ALTER TABLE statement gets called with? It might not matter much in this context but we should always use Rails' `Time.zone.now`. It's a bit fuzzy in my memory but I remember `Time.now` was a no. What I do know is that without `zone` is the only way to include time-zone information. According to https://thoughtbot.com/blog/its-about-time-zones we should let AR do the UTC conversion.